src/Workspaces/CSharp/Portable/Execution/CSharpOptionsSerializationService.cs

```diff
@@ -23,6 +21,7 @@ public override void WriteTo(CompilationOptions options, ObjectWriter writer, Ca
             var csharpOptions = (CSharpCompilationOptions)options;
             writer.WriteValue(csharpOptions.Usings.ToArray());
             writer.WriteBoolean(csharpOptions.AllowUnsafe);
+            writer.WriteByte((byte)csharpOptions.NullableContextOptions);
```

Comment:
Whenever the serialization format is updated, is there some version number which should be incremented?

Reply:
serialization format version number is needed if one is used for persistence. otherwise, version is not needed.